test/source/tests/compose.ts

```diff
@@ -236,6 +236,42 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       })
     );
 
+    test(
+      'user@disallow-password-message-terms.flowcrypt.test - disallow password  protected message terms',
+      testWithBrowser(async (t, browser) => {
+        const acct = 'user@disallow-password-message-terms.flowcrypt.test';
+        const rules = getKeyManagerAutogenRules(t.context.urls!.port!);
+        const disallowedPasswordMessageErrorText = 'Password-protected messages are disabled';
```

Comment:
let's add support for links here too, forgot to mention it in the issue description, thanks!

Reply:
Will the error text only contain links that the app should automatically detect and replace with <a> tags?
Or will it already include HTML tags (like <a href="...">)?

Reply:
We should auto-detect links, probably using `linkifyjs` will make it simple